lib/matplotlib/cbook.py

```diff
@@ -527,11 +545,7 @@ def process(self, s, *args, **kwargs):
         """
         if s in self.callbacks:
             for cid, proxy in list(six.iteritems(self.callbacks[s])):
-                # Clean out dead references
-                if proxy.inst is not None and proxy.inst() is None:
-                    del self.callbacks[s][cid]
-                else:
-                    proxy(*args, **kwargs)
+                proxy(*args, **kwargs)
```

Comment:
*cough* ``ReferenceError``.

And I think I should make ``remove_proxy `` private, a la ``_remove_proxy`` as we only every use the concept of proxies internally.